.changeset/metal-parrots-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react": patch
```

Comment:
I think it is worth a minor bump?

Reply:
Yeah, `ui-react-geo` needs to be here as a major release
Will add that